lib/Util/Notices/BlackFridayNotice.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace MailPoet\Util\Notices;
+
+use MailPoet\Models\Subscriber;
+use MailPoet\WP\Functions as WPFunctions;
+use MailPoet\WP\Notice as WPNotice;
+
+class BlackFridayNotice {
+
+  const OPTION_NAME = 'dismissed-black-friday-notice';
+  const DISMISS_NOTICE_TIMEOUT_SECONDS = 2592000; // 30 days
+
+  function init($should_display) {
+    $should_display = $should_display
+      && (time() <= strtotime('2019-11-30 23:59:59'))
```

Comment:
@pavel-mailpoet Should we not use the current date/time of the WordPress install itself and check against that using the core function? https://codex.wordpress.org/Function_Reference/current_time

It'd ensure there can be no bugs with various date/time formats. What do you think?

Reply:
@codemonkey-jack The time format is not an issue here, because both `time` and `strtotime` return the same format. See here:

- https://www.php.net/manual/en/function.time.php
- https://www.php.net/manual/en/function.strtotime.php